frontend/src/tests/components/admin/AdminDashboard.spec.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { shallow } from "enzyme";
+import { AdminDashboard } from "@/components/admin/AdminDashboard";
+
+describe("AdminDashboard", () => {
+  it("renders properly", () => {
```

Comment:
I'd really like to see us use our snapshot tests to validate that things are rendering "properly." Not in scope of this PR.

Reply:
let's chat about this sometime this sprint!